storage/src/mmr/bitmap.rs

```diff
@@ -147,13 +147,19 @@ impl<H: CHasher> Bitmap<H> {
         }
     }
 
-    /// Convert a bit offset into a bit mask.
-    fn mask_for(bit_offset: u64) -> u8 {
+    /// Convert a bit offset into a bit mask for the byte containing that bit.
+    pub(crate) fn mask_for(bit_offset: u64) -> u8 {
         1 << (bit_offset % 8)
     }
 
+    /// Convert a bit offset into the offset of the byte within a chunk containing the bit.
+    #[allow(dead_code)] // Remove when we start using this outside the test module.
+    pub(crate) fn chunk_byte_offset(bit_offset: u64) -> usize {
```

Comment:
If i'm not mistaken, these small functions can be `#[inline]`?

Reply:
added inline for these small helpers.